packer/linux/scripts/install-buildkite-test-splitter.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+case $(uname -m) in
+x86_64) ARCH=amd64 ;;
+aarch64) ARCH=arm64 ;;
+*) ARCH=unknown ;;
```

Comment:
We don't currently have an unknown arch file being created. Can I snip this or should be we building one of those?

Reply:
If `$(uname -m)` doesn't match a case, currently the script will fail when `curl` receives a 404 Not Found. If this line is snipped then the script will fail on the same line with an undefined variable error instead. Which would you prefer to have when trawling through CloudWatch logs trying to diagnose problems?

Reply:
Turns out the buildkite-agent install wont work if the architecture is unknown also, since https://github.com/buildkite/elastic-ci-stack-for-aws/blob/main/packer/linux/scripts/install-buildkite-agent.sh#L21 this will resolve to https://download.buildkite.com/agent/stable/3.74.0/buildkite-agent-linux-unknown which doesn't exist.